pom.xml

```diff
@@ -29,9 +30,9 @@
     <artifactId>forgerock-parent</artifactId>
     <version>2.0.7</version>
```

Comment:
@Kortanul Should we perhaps attempt to change the version and group / artifact id (I realize that will cause breakage)? Changing a release artifact feel so....dirty.....